app/cli/cmd/attestation_init.go

```diff
@@ -50,7 +51,7 @@ func newAttestationInitCmd() *cobra.Command {
 			}
 
 			// Initialize it
-			attestationID, err := a.Run(cmd.Context(), contractRevision)
+			attestationID, err := a.Run(cmd.Context(), contractRevision, workflowName)
```

Comment:
could we detect if you are providing a robot-account and a name and fail by telling you that you can not use name (or that name is ignored) in that case?

Reply:
I've added warning message indicating it will be ignored.